src/components/SideNavBar/index.tsx

```diff
@@ -86,11 +86,13 @@ const SideNavBar = ({
     ? 'side-nav-about side-nav--open'
     : 'side-nav-about';
 
-  if (typeof document !== 'undefined') {
-    document.body.style.overflow = navOpen
-      ? OverflowTypes.hidden
-      : OverflowTypes.unset;
-  }
+  useEffect(() => {
+    if (typeof document === 'object' && document.body) {
```

Comment:
if document doesn't exist we know we are not in a browser env so I think the other check was ok as is. I'm not sure why you are adding this effect though.

You could do `typeof document.body === 'object'`

Reply:
I made this issue: https://github.com/nodejs/nodejs.dev/issues/2674